engine/execution/computation/computer/computer.go

```diff
@@ -27,7 +27,10 @@ import (
 	"github.com/onflow/flow-go/utils/logging"
 )
 
-const SystemChunkEventCollectionMaxSize = 256_000_000 // ~256MB
+const (
+	ReusableCadenceRuntimePoolSize    = 1000
```

Comment:
Maybe improve naming here and everywhere else: the pool is not of runtimes, but *environments*

Reply:
As I mentioned previously, I'm going to the runtime into ReusableCadenceRuntime object.  The naming will make more sense then

Reply:
Ah, I missed the comment above, makes sense. Maybe add a TODO